cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -82,6 +84,18 @@ var (
 	privacyTagRegex = regexp.MustCompile(`(\[Privacy\->)|(<\-Privacy\])`)
 )
 
+// OSChoices returns the supported OS identifiers, sorted.
+func OSChoices() []string {
```

Comment:
GetSortedOSIDs or similar. Choice is not specific enough. Also, might get confused with a field without Get